### PR TITLE
[pvr] pvr.wmc version 0.5.3

### PR DESCRIPTION
**Pull request by scarecrow420**
This version adds support for the new ChannelGroup, Recording and EPG fields added in Kodi 15.0 PVR API 1.9.5 and 1.9.6

### CODE REVIEW

**Comment by hudokkow** [comment]
If nobody objects I'll merge this tomorrow.

File: src/pvr2wmc.h [hunk 1]
```diff
@@ -1,25 +1,23 @@
-#pragma once
 /*
-*      Copyright (C) 2011 Pulse-Eight
-*      http://www.pulse-eight.com/
```
Comment (MartijnKaijser) [inline src/pvr2wmc.h]: is this allowed to be changed?
@opdenkamp ?

Comment (scarecrow420) [inline src/pvr2wmc.h]: I don't believe that pulse8 thing was accurate to begin with, I believe the
addon was first created from a skeleton of another addon and then gutted,
so the copyright headers were from the other addon.

I'm not aware of any code in that file that is anything other than what
krustyreturns and other contributors like myself have produced. Perhaps the
socket class may be from somewhere. I was more so responding to a comment
on the old repo that our file headers should be cleaned up and made
consistent

Comment (opdenkamp) [inline src/pvr2wmc.h]: I guess it was c+p'd from the pvr.demo add-on in that case, which I originally wrote as a demo for Pulse-Eight. This header doesn't contain any of that code any more, the others may. Will have to check that. If you don't want to wait for that, then you can just keep both the team and Pulse-Eight in the copyright ("portions copyright Pulse-Eight Limited")

Comment (scarecrow420) [inline src/pvr2wmc.h]: Ive had a look at the pvr.demo code and I cant see anything that makes me think we need the pulse-eight copyright info

Of the 5 files that previously had it where ive now removed it:

src/client.h
header file, only contains declarations

src/client.cpp
boilerplate xbmc addon code that calls into pvr2wmc class

src/clientversion.h
Only contains the version string

src/pvr2wmc.h
Lars already confirmed this one is OK

src/pvr2wmc.cpp
I cant see anything problematic here, it really just sends backend calls and populates the XBMC structures

**Comment by hudokkow** [comment]
Addons with Pulse-Eight copyright:
> pvr.iptvsimple/src/client.cpp: *      Copyright (C) 2011 Pulse-Eight
pvr.iptvsimple/src/PVRIptvData.cpp: *      Copyright (C) 2011 Pulse-Eight
pvr.iptvsimple/src/client.h: *      Copyright (C) 2011 Pulse-Eight
pvr.iptvsimple/src/PVRIptvData.h: *      Copyright (C) 2011 Pulse-Eight
pvr.filmon/src/client.cpp: *      Copyright (C) 2011 Pulse-Eight
pvr.filmon/src/client.h: *      Copyright (C) 2011 Pulse-Eight
pvr.filmon/src/PVRFilmonData.h: *      Copyright (C) 2011 Pulse-Eight
pvr.filmon/src/PVRFilmonData.cpp: *      Copyright (C) 2011 Pulse-Eight
pvr.wmc/src/clientversion.h: *      Copyright (C) 2011 Pulse-Eight
pvr.wmc/src/client.cpp:*      Copyright (C) 2011 Pulse-Eight
pvr.wmc/src/client.h: *      Copyright (C) 2011 Pulse-Eight
pvr.wmc/src/pvr2wmc.h:*      Copyright (C) 2011 Pulse-Eight
pvr.wmc/src/pvr2wmc.cpp:*      Copyright (C) 2011 Pulse-Eight
pvr.demo/src/PVRDemoData.cpp: *      Copyright (C) 2011 Pulse-Eight
pvr.demo/src/client.cpp: *      Copyright (C) 2011 Pulse-Eight
pvr.demo/src/client.h: *      Copyright (C) 2011 Pulse-Eight
pvr.demo/src/PVRDemoData.h: *      Copyright (C) 2011 Pulse-Eight
pvr.demo/addon/addon.xml.in:  provider-name="Pulse-Eight Ltd.">

**Comment by opdenkamp** [comment]
As I said, I originally wrote the demo add-on for Pulse-Eight. The iptv add-on is the same as the demo one, with some additions that don't belong in the demo add-on.

The others have c+p'd some files from pvr.demo

**Comment by MilhouseVH** [comment]
Could this PR also bump addon.xml with xbmc.pvr 1.9.6?

**Comment by scarecrow420** [comment]
Thanks Milhouse, In my dev environment it seems that the addon automatically returns the latest API version, regardless of what the addon.xml says.  (not sure how/when the API version specified in addon.xml is used/checked).  But ive now updated that to 1.9.6 as well

@opdenkamp are we able to merge this?  As per above i checked all the copyright headers and im certain nothing in here has any relevance to historic PulseEight stuff

**Comment by scarecrow420** [comment]
ping @opdenkamp  @hudokkow it's been 2 weeks, can we please merge this PR?

**Pull request merged**